civihr_default_theme/assets/partials/myleave.html

```diff
@@ -0,0 +1,39 @@
+<div id="myleave_block">
```

Comment:
Do you actually need this? Given that you are using this only for applying the padding, can't you use one of the elements that Drupal already provides you? Like `.content` or `.main-container` or the like

If you indeed need this, then please follow the same dasherized naming convention: `.my-leave-block`